include/polycap.h

```diff
@@ -95,10 +98,11 @@ void polycap_rng_free(polycap_rng *rng);
 // construct a new polycap_photon with its initial position, direction, electric field vector and energy
 polycap_photon* polycap_photon_new(
 	polycap_rng *rng,
-	polycap_vector3 start_coords[3],
+	polycap_vector3 start_coords[3], //is the [3] relevant? Should just supply one start_coords structure of type polycap_vector3, not 3 such structures...
```

Comment:
Technisch vraagje. Moet die [3] hier wel staan? Immers `polycap_vector3 start_coords` op zich zou toch al een structure definieren met 3 delen (start_coords.x, .y en .z).
Zoals het er nu staat zou ik denken dat je een array van 3 structures maakt, dus start_coords[0].x [1].x [2].x, [0].y [1].y ... wat niet is wat we willen, toch?

Reply:
Nope die mag weg. Good catch.